### PR TITLE
add appveyor ci

### PR DESCRIPTION
**Pull request by q4a**
You will need to register on appveyor.com to add github project and see builds.

### CODE REVIEW

**Comment by Margen67** [comment]
There's already [GitHub Actions](https://github.com/Force67/ps4delta/actions) for CI.

**Comment by q4a** [comment]
I talked with @Force67 yesterday. He said: "more than 1 ci system isnt a bad thing to have"

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+configuration:
+- Release
+- Debug
+
+platform:
+- x64
+#- x86 # by design PS4Delta can't support x86 systems
+
+install:
+- cmd: |
```
Comment (Margen67) [inline appveyor.yml]: cmd is the default shell so you can just do `- |`

Comment (q4a) [reply appveyor.yml]: Thx for the tip. Done. Works well:
https://ci.appveyor.com/project/q4a/ps4delta/builds/30503622

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    git submodule update --init --recursive
+    set QT_DIR=C:\Qt\5.14.0\msvc2017_64
+    pmake.bat
+
+build:
+  project: build/PS4Delta.sln
+  parallel: true
+  verbosity: minimal
+
```
Comment (Margen67) [inline appveyor.yml:29]: If you want artifacts you'll need to add something like these:
```
after_build:
- 7z a Artifacts.zip .\build\bin\*

artifacts:
- path: Artifacts.zip
```

Comment (q4a) [reply appveyor.yml:29]: Done.

Comment (Margen67) [reply appveyor.yml:29]: artifacts should be put at the bottom.

Comment (q4a) [reply appveyor.yml:29]: I know, but here is no error, and for the future I'd like to do matrix with Windows and Linux parts and single `test: off` like here:
https://github.com/OpenXRay/xray-16/blob/xd_dev/appveyor.yml#L29

Another way: I can remove `test: off`, but I'll get such line:
`Discovering tests...OK`
Like this: https://ci.appveyor.com/project/q4a/ps4delta/builds/30504016/job/2adlynindhlx3ki9#L119

Comment (Margen67) [reply appveyor.yml:29]: Oh, ok

**Pull request merged**